src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap ;
+  gap: 1px;
+  align-items: center;
+  justify-content: center;
+
+  @for $i from 28 through 31 {
+    &--month-length-#{$i}
+    &__item:nth-last-of-type(-n + #{31 - $i}) {
+      display: none;
+    }
+  }
+
+  @each $day, $n in $week {
+    &--start-day-#{$day} {
+      :first-child {
+        margin-left: 606px;
```

Comment:
why 606? calculate this value based on other values (use scss variables)
you should use different margin for each start-day